src/bikeracks/templates/base.html

```diff
@@ -199,10 +176,10 @@ <h5 class="modal-title" id="removalModalTitle">Suggest removal</h5>
               <div class="modal-body">
                   <div class="dropdown">
                       
-                      <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                      <button class="btn btn-secondary dropdown-toggle removal-btn" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
It kind of looks weird that this is a button and not a normal dropdown (text entry + arrow) like everywhere else on the internet, e.g. https://jqueryui.com/selectmenu/

Reply:
Another option would be radio buttons + "other" with free text entry.

- Doesn't exist
- Duplicate
- Other: [_______________]